src/views/HomeView/index.js

```diff
@@ -3,7 +3,7 @@ import React, { PropTypes } from 'react'
 import { connect } from 'react-redux'
 import { increment, doubleAsync } from '../../redux/modules/counter'
 import DuckImage from './Duck.jpg'
-import classes from './HomeView.scss'
+import classes from './style.scss'
```

Comment:
Ideally BEM is solving the issue of having the CSS in the global namespace (and alot of more things :P), but It's exactly what you aim with CSS Modules, so... basically don't need to use any notation.

and @davezuko is right, there's no way of do a convention that likes to everyone! :baby_chick: